### DISCUSSION TITLE
Can I used grid without SASS?

### DISCUSSION THREAD

**Discussion by mylastore** [twbs/bootstrap#38483] [General] [answered]
I don't like SASS. Is there a pre-compiled version of Bootstrap 5.1 with grid enabled?

**Comment by julien-deramond** [comment, marked as answer]
Hey @mylastore,
There's no pre-compiled version of Bootstrap with grid enabled.
CSS Grid can be enabled only with Sass by setting `$enable-cssgrid: true` at the moment.
